test/test_extended_models.py

```diff
@@ -121,6 +121,88 @@ def get_models_from_module(module):
     assert a == b
 
 
+@pytest.mark.parametrize(
+    "module", [models, models.detection, models.quantization, models.segmentation, models.video, models.optical_flow]
```

Comment:
This parametrization isn't really adding a lot of relevant testing, because resnet and alexnet don't exist for the detection, segmentation, or video modules. I think it's fine to remove this parametrization and only test with the default `module=None`. Both logics (module vs filters) are completely de-coupled anyway.